engine/installation/binaries.md

```diff
@@ -81,37 +81,54 @@ exhibit unexpected behaviour.
 ## Get the Docker Engine binaries
 
 You can download either the latest release binaries or a specific version. To get
-the list of stable release version numbers from GitHub, view the `docker/docker`
+the list of stable releases and release candidates from GitHub, view the `docker/docker`
 [releases page](https://github.com/docker/docker/releases). You can get the MD5
-and SHA256 hashes by appending .md5 and .sha256 to the URLs respectively
+and SHA256 hashes by appending .md5 and .sha256 to the URLs respectively.
 
+A group of download links is included at the bottom of the release notes for
+a given version of Docker. You can use these links to download the source code
+archive for that release, as well as binaries for supported platforms and
+static binaries for unsupported Linux platforms. Use the links listed in the
+Downloads section to download the appropriate binaries.
 
-### Get the Linux binaries
+### Limitations of Windows and macOS binaries
 
-To download the latest version for Linux, use the
-following URLs:
-
-    https://get.docker.com/builds/Linux/i386/docker-latest.tgz
-
-    https://get.docker.com/builds/Linux/x86_64/docker-latest.tgz
-
-To download a specific version for Linux, use the
-following URL patterns:
-
-    https://get.docker.com/builds/Linux/i386/docker-<version>.tgz
-
-    https://get.docker.com/builds/Linux/x86_64/docker-<version>.tgz
-
-For example:
+You can only download the Windows binary for version `1.9.1` onwards.
+Moreover, the 32-bit (`i386`) binary is only a client, you cannot use it to
+run the `docker` daemon. The 64-bit binary (`x86_64`) is both a client and
+daemon.
```

Comment:
I think it's more accurate to say something like "The "i386" download contains a 32bit client-only binary. The "x86_64" download contains both client and daemon binaries for 64bit Windows Server 2016 and Windows 10 systems."